### ISSUE TITLE
Docs for subscribe

### ISSUE THREAD

**Issue by binhums** [machty/ember-concurrency#96]
**TLDR** Is subscribe intended to be used publicly and if so can we set up some docs and examples?

I found the subscribe method being used to capture keyboard events in the ember-concurrency presentation:

https://github.com/machty/ec-prezo/blob/master/app/application/controller.js#L108

```js
  keyboardListener: task(function * () {
    yield subscribe(events(Ember.$(document.body), 'keydown'), function * (ev) {
      ...
    });
  }).on('init'),
```

I like this pattern and I assume it is motivated by the need to destroy the subscription when its parent is destroyed.

I couldn't see information about `subscribe` in the API docs and in the end I think I got what I needed from the unit tests. Are we promoting the use of this feature?

If this is something we want to promote there are some issues with recent versions of Ember I can report separately.

**Comment by machty** [comment]
There's a handful of experimental e-c API, and I'd consider subscribe part of it. I believe you understand its intent and use cases. At one point, it was also intended to bridge the API gap with observables (maybe it still works?).

Either way, part of the reason I don't want to advertise it is because there are some other features I want to land that have architectural implications that are easier to land before elaborating upon the `subscribe` feature.

**Comment by binhums** [comment]
OK thanks for the clarification, feel free to close this issue. Do you want me to report the issues I'm finding with subcribe 2.9.1?

**Comment by machty** [comment]
Sure.
